sd_jwt/operations.py

```diff
@@ -0,0 +1,253 @@
+import datetime
+import logging
+import random
+from base64 import urlsafe_b64decode, urlsafe_b64encode
+from hashlib import sha256
+from json import dumps, loads
+from secrets import compare_digest
+from typing import Union
+
+from jwcrypto.jws import JWS
+from jwcrypto.jwk import JWK
+
+from sd_jwt.walk import by_structure as walk_by_structure
+from sd_jwt.demo_settings import (
```

Comment:
This is the only pending thing to do asap, because if we submit a custom settings, well, the paramenter MUST be taken by this latter

anyway it would be better to start a revision and merge in the main branch to prevent other misalignment with the ingoing changes, I'll do this refactor asap

Reply:
then I did in this way
https://github.com/oauthstuff/draft-selective-disclosure-jwt/pull/61/commits/71fdac35404c0127c21fcce9fbd3b386b932a434